src/state/action-creators.js

```diff
@@ -286,6 +296,7 @@ export const drop = () =>
       droppableScrollDiff: scrollDiff.droppable,
       windowScrollDiff: scrollDiff.window,
       draggables: state.dimension.draggable,
+      axis: destinationDroppable ? destinationDroppable.axis : null,
```

Comment:
add test to get-new-home-client-offset to ensure that it returns the default when null is passed in